cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java

```diff
@@ -247,6 +265,10 @@ public static Schema parseJson(Reader reader) throws IOException {
     return SCHEMA_TYPE_ADAPTER.fromJson(reader);
   }
 
+  private Object writeReplace() throws ObjectStreamException {
```

Comment:
Where is this being used?

Reply:
This is part of Java serialization specification: https://docs.oracle.com/javase/7/docs/api/java/io/Serializable.html

Reply:
+1, don't quite understand how SerializedForm is used.

Reply:
Clarified in javadoc